Documentation/ExtensionArchitecture/Extbase/Introduction/Index.rst

```diff
@@ -0,0 +1,109 @@
+.. include:: /Includes.rst.txt
+.. index::
+   pair: Extension development; Extbase introduction
+
+.. _extbase-introduction:
+
+====================
+Extbase introduction
+====================
+
+.. todo:: Refer to a good example, e.g. tea, blog_example?
+
+.. todo:: Add tips how to get more information or help
+
+What is Extbase?
+================
+
+Extbase is provided via a TYPO3 system extension (typo3-cms/extbase). Extbase is
+a framework within the TYPO3 core. The framework is based on `MVC
+(model-view-controller) <https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller>`__
+and uses
+`ORM (object relational modeling) <https://en.wikipedia.org/wiki/Object%E2%80%93relational_mapping>`__.
+
+Extbase can be and is often used in combination with the Fluid templating engine,
+but Fluid **can** also be used without Extbase. All things that can be done with
+Extbase, can also be done with core native functionality.
+Extbase is not a prerequisite for extension development. In most cases
+non-Extbase means writing more code. In many cases the performance will be
+better with non-Extbase.
+
+When Extbase was released, it was introduced as the modern way to program
+extensions and the "old" way (pibase) was propagated as outdated. When we look
+at this today, it is not entirely true: Extbase is a good fit for some specific
+types of extensions and there are always alternatives. For some use cases it
+is not a good fit at all and the extension can be developed without Extbase.
+
+Thus, many things, such as backend modules or plugins can be done "the Extbase
+way" or "the core way". This is a design decision, the extension developer must
+make for the specific use case.
+
+.. important::
+
+   Extbase is one of many options in the "toolbox" of TYPO3 extension development.
+
+   TYPO3 is flexible, often provides several approaches for one thing and it is
+   up to the extension developer to make an informed choice.
+
+Characteristics of Extbase
+==========================
+
+Conventions
+-----------
+
+Extbase uses the principle "Convention over configuration". This means that
```

Comment:
I have heard some Core developer arguing this. Often it is Convention. fullstop. and it is not possible to override the convention with configuration. Maybe it is also not the first most important element`?

Reply:
I removed the entire section. It was weird anyway to have the convention part as only entry in the section. I started writing that and it was all I could come up with at the moment. MVC and ORM is already mentioned above. Probably we do not need to go so much into detail on this page because that may be explained elsewhere or we can still add it later.